cf-templates/common/microservice.yml

```diff
@@ -6,13 +6,13 @@ Parameters:
     Description: Repository for getting the container
     Type: String
   ContainerName:
-    Description: Name for container used in various places
+    Description: Name  for container used in various places
```

Comment:
Why has these templates been modified? I don't think we should touch these templates

Reply:
i just modified the description  string to check the updation of version is working or not. i also removed the modification.